monitor.go

```diff
@@ -78,7 +78,7 @@ func monitorClusters(cloud *Cloud, filename string, destroyAfter time.Duration)
 	maybeAddVMs(cloud.NoExpiration)
 	maybeAddVMs(cloud.BadNetwork)
 	if len(badVMs) > 0 {
-		if err := deleteVMs(badVMs.Names()); err != nil {
+		if err := deleteVMs(badVMs.Names(), badVMs.Zones()); err != nil {
```

Comment:
You'll need to update the roachprod binary on the shared machine. See https://github.com/cockroachlabs/production/tree/master/shared
Just stick a linux binary in the binaries subdirectory, then `crl-put shared 1  binaries/roachprod roachprod`. The cron job will just pick it up.